mcs/class/System.Web/System.Web/HttpApplication.cs

```diff
@@ -209,6 +214,13 @@ internal void InitOnce (bool full_init)
 					if (context == null)
 						context = HttpContext.Current;
 					HttpModuleCollection coll = modules.LoadModules (this);
+
+					HttpModuleCollection dynMods = this.CreateDynamicModules ();
+
+					for(int i = 0; i < dynMods.Count; i++) {
+						coll.AddModule(dynMods.GetKey (i), dynMods.Get (i));
+					}
+
```

Comment:
Please reformat the block according to the Mono formatting guidelines (spaces before method calls)